File: benchmark/results/java-multilang/run-002/b-baseline/output/com/metaengine/demo/shipping/services/ShipmentRepository.java
package com.metaengine.demo.shipping.services;

import java.util.List;
import com.metaengine.demo.shipping.aggregates.Shipment;

/** ShipmentRepository in the shipping domain. */
public class ShipmentRepository {
    /** Create a new Shipment. */
    public Shipment create(Shipment input) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Find a Shipment by id. Returns null if not found. */
    public Shipment findById(String id) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** List Shipments up to limit. */
    public List<Shipment> list(int limit) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Delete a Shipment by id. */
    public void delete(String id) {
        throw new UnsupportedOperationException("not implemented");
    }
}

File: benchmark/results/python-multilang/run-001/b-baseline/output/metaengine_demo/shipping/services/shipment_repository.py
"""ShipmentRepository for the shipping domain."""
from typing import Optional

from metaengine_demo.shipping.aggregates.shipment import Shipment


class ShipmentRepository:
    """ShipmentRepository domain service."""

    def create(self, input: Optional[Shipment]) -> Shipment:
        raise NotImplementedError("not implemented")

    def findById(self, id: str) -> Optional[Shipment]:
        raise NotImplementedError("not implemented")

    def list(self, limit: float) -> list[Shipment]:
        raise NotImplementedError("not implemented")

    def delete(self, id: str) -> None:
        raise NotImplementedError("not implemented")
